pkgs/applications/altcoins/polkadot/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkgconfig
+, openssl
+}:
+
+rustPlatform.buildRustPackage rec {
+  name = "polkadot-${version}";
+  version = "0.2.17";
+
+  src = fetchFromGitHub {
+    owner = "paritytech";
+    repo = "substrate";
```

Comment:
This is confusing to me. This appears to be an entirely different repo than polkadot. What's going on?
Also, the rev looks to be different from the version string above (even though you're using rec, implying that you intended to reuse the version string?)

Can you explain what's going on so I can understand it?

Reply:
Okay, set revision.